server/api/batch_inventory.py

```diff
@@ -301,6 +301,57 @@ def process():
         session.commit()
 
 
+@api.route(
+    "/election/<election_id>/jurisdiction/<jurisdiction_id>/batch-inventory/system-type",
+    methods=["PUT"],
+)
+@restrict_access([UserType.JURISDICTION_ADMIN])
+def set_batch_inventory_system_type(
+    election: Election, jurisdiction: Jurisdiction  # pylint: disable=unused-argument
+):
+    system_type = request.get_json()["systemType"]
+    if system_type is None:
+        raise BadRequest("Missing systemType param")
+    if system_type not in [CvrFileType.DOMINION, CvrFileType.ESS]:
+        raise BadRequest(f"Unrecognized systemType param: {system_type}")
+
+    batch_inventory_data = BatchInventoryData.query.get(jurisdiction.id)
+    if not batch_inventory_data:
+        batch_inventory_data = BatchInventoryData(jurisdiction_id=jurisdiction.id)
+        db_session.add(batch_inventory_data)
+
+    batch_inventory_data.system_type = system_type
+
+    # Clear dependent data
+    if batch_inventory_data.cvr_file_id:
+        File.query.filter_by(id=batch_inventory_data.cvr_file_id).delete()
```

Comment:
This command not only deletes the referenced file record but also clears the `batch_inventory_data.cvr_file_id` field, given the DB schema:

https://github.com/votingworks/arlo/blob/36c7597b8eae01502831ef302d2a6b14321c6f64/server/models.py#L1019